ui/job-view/pushes/RevisionList.jsx

```diff
@@ -40,7 +40,12 @@ RevisionList.propTypes = {
 export function MoreRevisionsLink(props) {
   return (
     <li>
-      <a href={props.href} target="_blank" rel="noopener noreferrer">
+      <a
+        href={props.href}
+        target="_blank"
+        rel="noopener noreferrer"
+        title="show more revisions"
```

Comment:
Did you mean to put `title` on the icon?

Reply:
removed